### PR TITLE
More bug fixes (also working on overflow in CardCell)

### PR DESCRIPTION
**Pull request by matthewcoufal**

### CODE REVIEW

File: Clicker/ViewControllers/CardController.swift [hunk 1]
```diff
@@ -102,25 +104,35 @@ class CardController: UIViewController {
...
-        countLabel.clipsToBounds = true
-        countLabel.layer.cornerRadius = 0.5 * min(countLabelHeight, countLabelWidth)
+        countLabel.adjustsFontSizeToFitWidth = true
         updateCountLabelText(with: 0)
         view.addSubview(countLabel)
         
-        countLabel.snp.makeConstraints { make in
-            make.top.equalTo(view.safeAreaLayoutGuide.snp.top).offset(23)
+        countLabelBackgroundView.snp.makeConstraints { make in
+            make.top.equalTo(view.safeAreaLayoutGuide.snp.top).offset(24)
```
Comment (jschluger) [inline Clicker/ViewControllers/CardController.swift]: Make this a constant? for consistency

File: Clicker/Views/Cards/CardCell.swift [hunk 2]
```diff
@@ -218,7 +218,7 @@ extension CardCell: ListAdapterDataSource {
         }
         objects.append(separatorLineModel)
         objects.append(pollOptionsModel)
-        objects.append(bottomHamburgerCardModel)
+     //   objects.append(bottomHamburgerCardModel)
```
Comment (jschluger) [inline Clicker/Views/Cards/CardCell.swift]: Remove this if we aren't using it.

File: Clicker/Views/Cards/PollOptionsCell.swift [hunk 3]
```diff
@@ -50,14 +57,23 @@ class PollOptionsCell: UICollectionViewCell {
         adapter = ListAdapter(updater: updater, viewController: nil)
         adapter.collectionView = collectionView
         adapter.dataSource = self
+        adapter.scrollViewDelegate = self
+    }
+    
+    func scrollViewDidScroll(_ scrollView: UIScrollView) {
+        let diff = collectionView.contentSize.height - bounds.height - scrollView.contentOffset.y
+        if diff < 10 {
+
```
Comment (jschluger) [inline Clicker/Views/Cards/PollOptionsCell.swift]: What's going on here? This looks not finished...

Comment (matthewcoufal) [reply Clicker/Views/Cards/PollOptionsCell.swift]: Not done with that yet. I'll get it done before the meeting tomorrow.

File: Clicker/ViewControllers/CardController.swift [hunk 4]
```diff
@@ -176,7 +188,7 @@ class CardController: UIViewController {
     
     func updateCountLabelText(with index: Int) {
         let total = pollsDateModel.polls.count
-        countLabel.attributedText = getCountLabelAttributedString("\(index + 1)/\(total)")
+        countLabel.attributedText = getCountLabelAttributedString("\(index + 1) / \(total)")
```
[the remark this answers is not in the document]
Comment (matthewcoufal) [reply Clicker/ViewControllers/CardController.swift]: Yes. He approves of the space.

**Pull request merged**